cmd/client-admin.go

```diff
@@ -92,7 +162,7 @@ func NewAdminFactory() func(config *Config) (*madmin.AdminClient, *probe.Error)
 				tlsConfig.InsecureSkipVerify = true
 			}
 
-			var transport http.RoundTripper = &http.Transport{
+			transport = &http.Transport{
```

Comment:
this is duplicate code, it's better to move this above where you start the credentials chain